controls/V-72219.rb

```diff
@@ -1,7 +1,59 @@
 # encoding: utf-8
 #
 
-# TODO we need to write tests on this.
+firewalld_services_deny = input(
+  'firewalld_services_deny',
+  value: {
+    # Examples (zone:[services])
+    # "public"=>['ssh', 'icmp'],
+    # "dmz"=>['http']   
+  },
+  description: "Services that firewalld should be configured to deny."
+)
+
+firewalld_hosts_deny = input(
+  'firewalld_hosts_deny',
+  value: [
+    # Example
+    # 'rule family="ipv4" source address="104.56.21.1/24" accept]'
+  ],
+  description: "Hosts that firewalld should be configured to deny."
+)
+
+firewalld_ports_deny = input(
+  'firewalld_ports_deny',
+  value: {
+    # Examples (zone:[ports])
+    #"public"=>['12345/tcp', '23456/tcp'],
+    # "dmz"=>['80/http', '22/ssh']
+  },
+  description: "Ports that firewalld should be configured to deny."
+)
+
+firewalld_zones = input(
+  'firewalld_zones',
+  value: [
+    # Examples
+    # 'drop',
+    # 'block',
+    # 'public'
+    # 'external',
+    # 'dmz',
+    # 'work',
+    # 'home',
+    # 'internal',
+    # 'trusted'
+  ]
+)
+
+iptable_rules = input(
```

Comment:
Should this be `iptables` instead of `iptable`?

Reply:
Just fixed the typo.

Reply:
@trevor-vaughan  Can we merge this control in now?